libs/sdk-ui-dashboard/src/model/store/filterContext/filterContextReducers.ts

```diff
@@ -264,30 +264,32 @@ const addAttributeFilter: FilterContextReducer<PayloadAction<IAddAttributeFilter
         title,
     } = action.payload;
 
+    // Filters are indexed just for attribute filters, if DateFilter is present should be always first item
+    const isDateFilterPresent = state.filterContextDefinition.filters.findIndex(isDashboardDateFilter) >= 0;
+
     const hasSelection = initialSelection && !attributeElementsIsEmpty(initialSelection);
 
     const isNegative = selectionMode !== "single" && (initialIsNegativeSelection || !hasSelection);
+    // If DateFilter is present we have to move index by 1 because index of filter is calculated just for AttributeFilers array
+    const attributeFilterIndex = isDateFilterPresent ? index + 1 : index;
```

Comment:
just curious, what happens when we have multiple date filters in the filter bar? Can you please double-check this case?